### PR TITLE
Update all gems (incl. jekyll to 4.1)

### LINKED ISSUE #127: migrate to jekyll 4
jekyll 4 seems to break the navigation menu. please check why and how to fix.

may be solved by jekyll 4.1 (compare [this comment on related issue](https://github.com/jekyll/jekyll/pull/7948#issuecomment-584132037)), but not sure whether this really is the problem since they are only warnings.

### PR DESCRIPTION
**Pull request by kimbtech**
Should fix #127.

 As @nicoa stated jekyll 4.1 seems to keep our menue at least on my pc.

### CODE REVIEW

File: .bundle/config [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+BUNDLE_PATH: "vendor/bundle"
```
Comment (nicoa) [inline .bundle/config:2]: why this?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,5 @@
 _site/
 .sass-cache/
 .jekyll-metadata
+.jekyll-cache/
+vendor/
```
Comment (nicoa) [inline .gitignore:5]: add newline

File: .bundle/config (hunk 1 above)
Comment (kimbtech) [reply .bundle/config]: Oh, I thought this file would be gitignored. Deleted it now.

File: .gitignore (hunk 2 above)
Comment (nicoa) [inline .gitignore:5]: not necessary but doesn't matter ;-)

**Review by nicoa — APPROVED** [review]

**Pull request merged**